### PR TITLE
Localization [some words are missing in Malayalam] in efiling

### PR DESCRIPTION
**Pull request by utpalBeehyv**
https://github.com/pucardotorg/dristi/issues/2261

## Release Notes

- **New Features**
	- Enhanced internationalization support across various components, allowing for dynamic translation of UI text based on user language preferences.
  
- **Bug Fixes**
	- Updated button labels and text elements to use localized keys, improving clarity and consistency in the user interface.

- **Refactor**
	- Streamlined rendering logic in the `SummonsOrderComponent` for better code maintainability while preserving functionality.

These updates improve user experience by ensuring that the application is more accessible and user-friendly in multiple languages.

### CODE REVIEW

**Review by vaibhavct — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/components/CustomDetailsCard.js [hunk 1]
```diff
@@ -10,50 +12,72 @@ const CustomDetailsCard = ({ header, subtext, serialNumber, subnote, style }) =>
...
+            fontSize: "16px",
+            fontWeight: 700,
+            lineHeight: "18.75px",
+            textAlign: "left",
+            color: "#77787B",
+            margin: "0px",
+          }}
+        >
+          {t(serialNumber)}
+        </p>
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/components/CustomDetailsCard.js:33]: No Need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/registration/Response.js [hunk 2]
```diff
@@ -40,15 +40,15 @@ const Response = ({ refetch, setHideBack }) => {
                   refetch();
                   history.push(`/${window?.contextPath}/citizen/dristi/home/file-case`);
                 }}
-                label={t("File a case")}
+                label={t("FILE_CASE")}
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/registration/Response.js:43]: Irrelevant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/EvidenceHeader.js [hunk 3]
```diff
@@ -175,13 +177,13 @@ const EvidenceHearingHeader = ({ hearing, caseData, filingNumber, setActiveTab,
             className={activeTab === "Transcript/Summary" ? "search-tab-head-selected" : "search-tab-head"}
             onClick={() => setActiveTab("Transcript/Summary")}
           >
-            {t("Transcript/Summary")}
+            {t("TRANSCRIPT_SUMMARY")}
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/EvidenceHeader.js:180]: Done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js [hunk 4]
```diff
@@ -17,42 +17,39 @@ const RenderDeliveryChannels = ({ partyDetails, deliveryChannels, handleCheckbox
...
-                  <h2>
-                    <strong>{t(channel.label)} to </strong>
-                  </h2>
+        {deliveryChannels.map((channel) => (
+          <div key={channel?.type}>
+            {Array.isArray(channel?.values) && channel?.values?.length > 0 && channel?.values[0] != null && (
+              <div>
+                <h2>
+                  <strong>{t(channel.label)} to </strong>
+                </h2>
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js:26]: No Need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js [hunk 5]
```diff
@@ -17,42 +17,39 @@ const RenderDeliveryChannels = ({ partyDetails, deliveryChannels, handleCheckbox
...
+                            ? value.address
+                            : `${value.locality}, ${value.city}, ${value.district}, ${value.pincode}`
+                          : value}
+                      </label>
+                    </div>
+                  ))}
+              </div>
+            )}
+          </div>
+        ))}
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js:52]: no Need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/EFilingCases.js [hunk 6]
```diff
@@ -2233,7 +2233,12 @@ function EFilingCases({ path }) {
                 {`${t(pageConfig.header)}`}
                 {pageConfig?.showOptionalInHeader && <span style={{ color: "#77787B", fontWeight: 100 }}>&nbsp;(optional)</span>}
                 {selected === "reviewCaseFile" && (
-                  <Button className="border-none dristi-font-bold" onButtonClick={handleViewCasePdf} label={t("View PDF")} variation={"secondary"} />
+                  <Button
+                    className="border-none dristi-font-bold"
+                    onButtonClick={handleViewCasePdf}
+                    label={t("CS_VIEW_PDF")}
+                    variation={"secondary"}
+                  />
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/EFilingCases.js:2241]: Irrelavant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/components/CustomDetailsCard.js [hunk 7]
```diff
@@ -10,50 +12,72 @@ const CustomDetailsCard = ({ header, subtext, serialNumber, subnote, style }) =>
...
+            fontFamily: "Roboto",
+            fontSize: "16px",
+            fontWeight: 700,
+            lineHeight: "18.75px",
+            textAlign: "left",
+            color: "#77787B",
+            margin: "0px",
+          }}
+        >
+          {t(serialNumber)}
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/components/CustomDetailsCard.js:32]: Ireelavant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/HearingSideCard.js [hunk 8]
```diff
@@ -12,10 +14,10 @@ const HearingSideCard = ({ hearingId, caseId, filingNumber }) => {
   return (
     <div className="hearing-side-card" style={{ width: "auto" }}>
       <div className="hearing-details">
-        <div className="hearing-title">Case Details</div>
+        <div className="hearing-title">{t("CASE_DETAILS")}</div>
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/HearingSideCard.js:17]: Will address later

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/CaseType.js [hunk 9]
```diff
@@ -308,64 +308,60 @@ function CaseType({ t }) {
...
-      { header: "Case Category", subtext: "Criminal" },
+      { header: "CASE_CATEGORY", subtext: "CS_CRIMINAL" },
       {
-        header: "Status / Act",
-        subtext: "Negotiable Instruments Act",
+        header: "CS_STATUS_ACT",
+        subtext: "CS_NIA",
       },
-      { header: "Section", subtext: "138" },
+      { header: "CS_SECTION", subtext: "138" },
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/CaseType.js:316]: Already Done with that name

**Pull request merged**